tests/integration_test.rs

```diff
@@ -60,7 +60,7 @@ fn different_numeric_types_assume_float() {
     let actual = json!({ "a": { "b": true }, "c": [true, null, 1] });
     let expected = json!({ "a": { "b": true }, "c": [true, null, 1.0] });
     let config = Config::new(CompareMode::Inclusive).numeric_mode(NumericMode::AssumeFloat);
-    assert_json_matches!(&actual, &expected, config.clone());
+    assert_json_matches!(actual, expected, config.clone());
```

Comment:
Is this a breaking change?

Reply:
No. The old version compiles, this is just a little simpler. I can test both if you like.